app/src/newdocs/content/SDKs.mdx

```diff
@@ -12,7 +12,18 @@ import { JavascriptExample } from './code/api.js'
 <ScrollableAnchor id="sdks-overview"><div>
 
 ## Overview
---content--
+There are both official and unofficial SDKs (Software Development Kits) available for building software that interacts
+with the Streamr platform. Using an SDK provides an effortless and reliable way to use Streamr from your software
+projects. An SDK hides the intricacies involved of interacting with Streamr behind an intuitive library API, allowing a
+developer to focus on their task at hand.
+
+
+Currently, we provide official SDKs for Javascript (browser and Node.js) and Java. There is also an experimental SDK for
+Python in the works. The range of features each of these SDKs provide may vary; be sure to check the documentation of
+a specific SDK for more information.
```

Comment:
A  feature matrix would be nice here. Here's a starting point, feel free to improve:


|                            | [JS](https://github.com/streamr-dev/streamr-client-javascript) | [Java](https://github.com/streamr-dev/streamr-client-java) | [Python](https://github.com/streamr-dev/streamr-client-python) |
| --                         | -- | --   | --     |
| Publish                    | ✅ | ✅   | ✅     |
| Subscribe                  | ✅ | ✅   | ✅     |
| Managing Streams           | ✅ | ✅   | ✅     |
| API Key Authentication     | ✅ | ✅   | ✅     |
| Ethereum Authentication    | ✅ | ✅   | ❌     |
| Duplicate Detection        | ✅ | ✅   | ✅     |
| Gap Filling                | ✅ | ✅   | ✅     |
| Message Signing            | ✅ | ✅   | ❌     |
| Message Verification       | ✅ | ✅   | ❌     |
| End-to-end Encryption      | ✅ | ✅   | ❌     |
| Requesting Historical Data | ✅ | ✅   | ❌     |